package.json

```diff
@@ -3,87 +3,59 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@babel/polyfill": "^7.12.1",
     "@babel/runtime": "^7.12.5",
     "@emotion/css": "^11.0.0",
-    "@emotion/react": "^11.1.1",
+    "@emotion/react": "^11.1.2",
     "@emotion/styled": "^11.0.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.32",
```

Comment:
A bunch of these dependencies aren't used by the (soon to be) published bundle, so I've moved them to devDeps.